app/assets/javascripts/mirador_widget_admin.js

```diff
@@ -16,6 +16,7 @@
     // that contains the value of the current text input
     function setupItemInputButtonEvents(block) {
       sourceLocationSubmit(block).on('click', function() {
+        clearError(block);
```

Comment:
🤔 wondering if this would be good to be placed in the `setupItemSubmittedListener` that way if other code needs to trigger the `item-submitted` event the error will be cleared automatically.  Not really sure if it matters at this point, so up to you!

Reply:
Sounds reasonable to me 👍